server/game/cards/locations/06/stormsend.js

```diff
@@ -0,0 +1,34 @@
+const DrawCard = require('../../../drawcard.js');
+
+class StormsEnd extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onDominanceDetermined: (event, winner) => this.controller === winner
+            },
+            cost: ability.costs.discardFactionPower(1),
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    numCards: 2,
+                    activePromptTitle: 'Select 2 characters to gain power',
+                    source: this,
+                    cardCondition: card => card.getType() === 'character',
+                    onSelect: (player, cards) => this.onSelect(player, cards)
+                });
+            }
+        });
+    }
+    
+    onSelect(player, cards) {
+        cards[0].modifyPower(1);
```

Comment:
I know you "know" there's only 2, but i still think a _.each would be beneficial here